### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by tlylt**
Hello,

Included a link to a website I made for ease of collating and viewing of advice from NUS seniors to freshmen.
Hope that's ok.

Thanks!

_(P.S. I guess this PR is not welcomed, will close with my apologies)_

### CODE REVIEW

**Comment by J0** [comment]
Hey @tlylt,

Disclaimer: I'm a senior and have no say as to whether this PR get approves.

First of all, kudos to you for taking the time to build an application! It is heartwarming that you took the time to build an application to improve the quality of the advice collation. It realiz=se it might feel disappointing that the team isn't replying you after you put in effort to build this app and I'm sorry if the lack of response made it seem like the PR wasn't welcome!

I don't think the team is intentionally not replying your PR --  coreteam is quite busy and there's a  good chance that they have missed your post so I suggest that you ping them again here: https://t.me/nushackers_chat

As a senior I do have a couple of suggestions/questions about the app w.r.t Github though:

1. As a senior, how can I verify that the contributions that have been made belong to who they claim to be without providing authentication. I am concerned about my words being misattributed.

2. As a senior, how can I delete my posts easily. Again, after some time I might find what I've said is no longer relevant and I might want to delete it.

I think the application looks great! But as a senior providing advice I would be a bit hesitant to use the app over Github rn simply because I'm quite lazy and I like what is already working for me. Other seniors might feel differently so you might want to talk to them first.

tl;dr I don't think your PR is being intentionally ignored so please don't let that discourage you but I'm also not sure that there'd be a strong incentive to move notes/advice off Github atm.

**Comment by tlylt** [comment]
> Hey @tlylt,
> 
> Disclaimer: I'm a senior and have no say as to whether this PR get approves.
> 
> First of all, kudos to you for taking the time to build an application! It is heartwarming that you took the time to build an application to improve the quality of the advice collation. It realize it might feel disappointing that the team isn't replying you after you put in effort to build this app and I'm sorry if the lack of response made it seem like the PR wasn't welcome!
> 
> I don't think the team is intentionally not replying your PR -- coreteam is quite busy and there's a good chance that they have missed your post so I suggest that you ping them on Telegram at the NUS Hackers Chat: [t.me/nushackers_chat](https://t.me/nushackers_chat) if you feel strongly about the app.
> 
> As a senior I do have a couple of suggestions/questions about the app w.r.t Github though:
> 
> 1. As a senior, how can I verify that the contributions that have been made belong to who they claim to be without providing authentication. I am concerned about my words being misattributed.
> 2. As a senior, how can I delete my posts easily. Again, after some time I might find what I've said is no longer relevant and I might want to delete it.
> 
> I think the application looks great! But as a senior providing advice I would be a bit hesitant to use the app over Github rn simply because I'm quite lazy and I like what is already working for me. Other seniors might feel differently so you might want to talk to them first.
> 
> tl;dr I don't think your PR is being intentionally ignored so please don't let that discourage you but I'm also not sure that there'd be a strong incentive to move notes/advice off Github atm.

Hey man, thank you for the thoughtful response :)
